web/source/kmwembedded.ts

```diff
@@ -78,11 +78,11 @@ namespace com.keyman.osk {
         this.keytip = new osk.embedded.KeyTip(window['oskCreateKeyPreview'], window['oskClearKeyPreview']);
       }
     };
+  }
```

Comment:
So all you did was remove the wait for fonts function out of `com.keyman.osk`  namespace?

Reply:
Not quite. I moved the `waitForFonts` function override up a level so it was not in the Android-specific code path.